### DISCUSSION TITLE
Assets - Not possible to add image

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#6123] [Help / Questions]
Hi,
  When I try to add image to a page, I have all the time this error when I click on the buton to add image or if I upload image.

![image](https://user-images.githubusercontent.com/6518973/216946861-ca447fc6-3ed7-4210-985a-2dff302d9563.png)

I'm logged as Administrator.  But I created a new group which has all the rights on the assets (read/write) and the Administrator is included in this group.

Thanks for your help

**Comment by NGPixel** [comment]
Are you uploading an image larger than the allowed size?

**Comment by ghost** [comment]
hi,The error ils show before i upload an image.ChristianLe 7 févr. 2023 00:50, Nicolas Giard ***@***.***> a écrit :
Are you uploading an image larger than the allowed size?
